dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/RegisteredService.java

```diff
@@ -26,9 +22,7 @@
  * {@code RegisteredService.of()}.
  */
 @SuppressWarnings("all")
-@ParametersAreNonnullByDefault
 @Generated({"Immutables.generator", "AbstractRegisteredService"})
```

Comment:
See discussion in PR #688 
https://github.com/lagom/lagom/pull/688/files/86244a9188dad011402ec4eb63a2f34167e80322#r112859236